script/hardhat/fork/govUpdateTest.ts

```diff
@@ -0,0 +1,325 @@
+import { impersonateAccount, loadFixture, mine, mineUpTo, setBalance } from "@nomicfoundation/hardhat-network-helpers";
+import { SignerWithAddress } from "@nomiclabs/hardhat-ethers/signers";
+import { expect } from "chai";
+import { parseUnits } from "ethers/lib/utils";
+import { ethers } from "hardhat";
+
+import { Comptroller, GovernorBravoDelegate, IAccessControlManager } from "../../../typechain";
+import { TimelockInterface } from "../../../typechain/contracts/Governance/GovernorAlpha2.sol";
+import { getCalldatas, setForkBlock } from "./vip-framework/utils";
+
+const ONE_HOUR = 3600; // 1 hour
+const SIX_HOURS = 6 * 3600; // 6 hours
```

Comment:
nit: could multiply the constant, which will be nice in the event the number of seconds in an hour changes 😝 
```suggestion
const SIX_HOURS = 6 * ONE_HOUR; // 6 hours
```